.circleci/config.yml

```diff
@@ -0,0 +1,117 @@
+version: 2.1
+
+orbs:
+  aws-ecr: circleci/aws-ecr@6.8.1
+  aws-cli: circleci/aws-cli@1.0.0
+  slack: circleci/slack@3.4.2
```

Comment:
Why is this here?

Reply:
`slack` doesn't need to be here, I'll remove it :)